From 671d3be6fa17e4ee53cef624b15b4a039bef90b4 Mon Sep 17 00:00:00 2001
From: Felix Radensky <felix.r@variscite.com>
Date: Tue, 10 Nov 2020 14:52:51 -0800
Subject: [PATCH 1/3] imx8m: Adjust struct dram_timing_info to match u-boot

ATF struct dram_timing_info holds DRAM configuration created by
u-boot. Variscite u-boot code added additional member to this structure.
Make ATF definition mirror the u-boot one.

Signed-off-by: Harshesh Valera <harshesh.v@variscite.com>
---
 plat/imx/imx8m/include/dram.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/plat/imx/imx8m/include/dram.h b/plat/imx/imx8m/include/dram.h
index 144316af9..22edb3bdb 100644
--- a/plat/imx/imx8m/include/dram.h
+++ b/plat/imx/imx8m/include/dram.h
@@ -31,6 +31,8 @@ struct dram_cfg_param {
 };
 
 struct dram_timing_info {
+	/* ddrphy firmware offset */
+	unsigned int ddrphy_fw_offset;
 	/* umctl2 config */
 	struct dram_cfg_param *ddrc_cfg;
 	unsigned int ddrc_cfg_num;
-- 
2.25.1

